go/saltpack/specs/saltpack_armor.md

```diff
@@ -0,0 +1,193 @@
+# SaltPack ASCII Armor Format
+
+Like PGP, SaltPack encrypted and signed messages are binary formats. We need to
+wrap them in an ASCII format, to let people paste them into websites and apps
+that handle text.
+
+The main building block of this format is a generic bytes-to-characters
+encoding, similar to Base64, which we call Just Numbers. We chose not to use
```

Comment:
50% just a standin name, like "sillypack", which I figured we'd change to something else. 50% because it really is just arithmetic base conversion.